src/guessWords/index.js

```diff
@@ -0,0 +1,3 @@
+import App from "./containers/App";
+
+export default App;
```

Comment:
It's OK to use any name while exporting as default but it could be better if you import it with another name.